src/modules/public/blacklistAccess/actions.test.ts

```diff
@@ -0,0 +1,25 @@
+import * as actions from './actions';
+import {
+    SEND_ACCESS_TOKEN_DATA,
+    SEND_ACCESS_TOKEN_ERROR,
+    SEND_ACCESS_TOKEN_FETCH,
+} from './constants';
+
+describe('Blacklist country actions', () => {
```

Comment:
1. Name blacklistAccess is confusing, rather it\'s whitelistAccess.
2. Rename ```Blacklist``` to ```Denylist``` and ```blacklistAccess``` to ```allowlistAccess```
According to replacement of problematic racist terminology (to avoid changing in the future)